### PR TITLE
second attempt to add Svelte TOC

### LINKED ISSUE #165: try @svelte-put/toc for TOC instead
#162 was a bust but @vnphanquang has [svelte-put.vnphanquang.com/docs/toc](https://t.co/aln2DKHTCr) which is promising

### PR DESCRIPTION
**Pull request by sw-yx**
closes #165

### CODE REVIEW

**Comment by sw-yx** [comment]
ouch. no bueno.

Screen Shot 2023-01-15 at 9 12 33 AM

**Comment by sw-yx** [comment]
i think netlify's deploy preview may have added extra js weight. i turned it off

**Comment by sw-yx** [comment]
yea confirmed i think it only adds about 1.2kb of js which is fine. sorry for false alarm @vnphanquang

image

**Comment by vnphanquang** [comment]
How's LCP and such? Is the performance drop still significant enough to impact UX?

**Comment by sw-yx** [comment]
no its fine, i even made some mild unrelated improvements

File: tailwind.config.cjs [hunk 1]
```diff
@@ -42,7 +42,7 @@ module.exports = {
 							textDecoration: 'underline !important'
 						},
 						a: {
-							color: '#3182ce',
+							color: '#2071ad',
```
Comment (sw-yx) [inline tailwind.config.cjs:45]: kept getting flagged in a11y review so darkened it a bit

**Pull request merged**